requirements-py3.txt

```diff
@@ -63,3 +63,4 @@ WebTest==2.0.33
 zope.deprecation==4.4.0
 zope.interface==4.6.0
 unidecode==1.1.1
+
```

Comment:
requirements-py3.txt should be removed.

Reply:
I tried, but apparently, the setup.py file is using this in installation for pipenv install -e (got an error saying the file couldn't be found)
We'll have to make changes in setup.py to be able to remove this file, and it would be best to do it in a second time.

I can include a comment in the package to specify this.

Reply:
Then we have to modify `setup.py`. This has to be done within this PR, having two requirement files is not a good, so we have to be sure that we can get rid of the old one, otherwise it don't make sense to go to pipenv.
One idea could be to modify `setup.py` to generate the requirements using `pipenv lock -r` command instead of reading the old requirement file.